release-notes/opensearch-neural-search.release-notes-2.10.0.0.md

```diff
@@ -4,6 +4,7 @@ Compatible with OpenSearch 2.10.0
 
 ### Features
 * Improved Hybrid Search relevancy by Score Normalization and Combination ([#241](https://github.com/opensearch-project/neural-search/pull/241/))
+* Support sparse semantic retrieval by introducing `sparse_encoding` ingest processor and query builder ([#333](https://github.com/opensearch-project/neural-search/pull/333))
 
 ### Enhancements
 * Changed format for hybrid query results to a single list of scores with delimiter ([#259](https://github.com/opensearch-project/neural-search/pull/259))
```

Comment:
as this is the first change in the neural search after 2.10 launch can you remove all other entries and just put the one you are adding.

Reply:
Sure, or maybe we can create a 2.11 release note and put this entry there

Reply:
Get it, will move the commit in this file to change log